include/common/mir/events/event_builders.h

```diff
@@ -200,8 +200,8 @@ EventUPtr make_touch_event(
 EventUPtr clone_event(MirEvent const& event);
 void set_window_id(MirEvent& event, int window_id);
 
+[[deprecated("Not meaningful: legacy of mirclient API")]]
 EventUPtr make_start_drag_and_drop_event(frontend::SurfaceId const& surface_id, std::vector<uint8_t> const& handle);
-void set_drag_and_drop_handle(MirEvent& event, std::vector<uint8_t> const& handle);
```

Comment:
I'm confused. This is in a public header and you mark the above function deprecated, so why are you allowed to delete this one?

Reply:
I'm not. Good  catch